terra-prod.json

```diff
@@ -41,6 +41,21 @@
     }
   },
   "profile_vars": {
+    "firecloud-org-terraform-sa": {
```

Comment:
Should this be setting the SA roles?

Reply:
Tried, but they need to be org-level so I did it manually. I'll circle back if it seems necessary.